### DISCUSSION TITLE
#8 8th Sprint - Software Architecture Documentation

### DISCUSSION THREAD

**Discussion by Annalytic-programming** [CUMGroup/AMOGUS#48] [Blog]
Hiya! Hi!

last week it was the [SAD's](https://github.com/CUMGroup/AMOGUS/blob/main/documents/architecture/SoftwareArchitectureDocumentation.md) turn,
some sort of Software Architecture Documentation we learn.
Wether view, service layer or the API,
everything we had to clarify.

Even a new service was implemented,
guess what, it's object oriented.
User service is what i'm talking about,
in our backend we now have a new route.

There was just little time for our programs
between studying and all the exams,
but things go well in our team
even our software architecture scheme.

we hope you like the blog this week
and could give you a tiny peek.
To make this poem throughly:
We wish you: **Yours truly**

CUMGroup

**Comment by N-Eisen** [comment]
You have chosen a very beautiful and creative display. Your SAD is good, but some points are (almost) not worked on.
Many greetings Framgang

**Reply by jas20202** [reply]
Well hello Framgang,

thank you for appreciating our creative display of our blog :D
Though back then it wasn't mandatory to have the SAD 100% finished and as far as we can remember all the necessary points were completed. But now the SAD should be completely done so if you'd like you can take a look ^^

We wish you a happy new year and a successfull practical phase 🎇 

Yours truly,
CUMGroup (Project AMOGUS)

**Comment by vans1646** [comment]
Hi CUMGroup,
I had a look at your SAD and was not sad. 
You filled almost everything and because (I think) the full document is not due this week, it's pretty good. I also enjoyed your new diagrams and can say that you did a good job!
Your rhyming style of blogging was also very refreshing.

Until next week!
Capyclue

**Reply by jas20202** [reply]
Hello Capyclue,

very nice to see you around here again \\(^°^)/

We are very glad you liked our SAD!
Just as we are happy to hear that you enjoyed our diagrams as well as our rhyming style.

We hope you have a happy new year and a successfull practical phase 🎇 

Yours truly,
CUMGroup (Project AMOGUS)

**Comment by TeamTuDu** [comment]
Hey CUMGroup,

clear and consize news 
very useful new features
impressive, not sad

Best regards
Team TuDu

**Reply by jas20202** [reply]
Dear team TuDu,

thank you for the short and precise comment ^^

We wish you a happy new year and a successfull practical phase 🎇 

Yours truly,
CUMGroup (Project AMOGUS)
